### PR TITLE
SQL: specify command to run the CLI on a remote machine without Elasticsearch

### PR DESCRIPTION
**Pull request by astefan**
Small docs improvement by specifying a Java command to run the SQL CLI on a machine where Elasticsearch is not installed on.

Addresses https://github.com/elastic/elasticsearch/issues/48729

### CODE REVIEW

File: docs/reference/sql/endpoints/cli.asciidoc [hunk 1]
```diff
@@ -47,3 +43,27 @@ James S.A. Corey |Leviathan Wakes     |561            |1306972800000
...
+
+or
+
+[source,bash]
+--------------------------------------------------
+$ ./java -cp [PATH_TO_CLI_JAR]/elasticsearch-sql-cli-[VERSION].jar org.elasticsearch.xpack.sql.cli.Cli https://some.server:9200
+--------------------------------------------------
+
+The jar name will be different for each Elasticsearch version, thus
+the generic `VERSION` specified in the example above. Also, if not running
```
Comment (matriv) [inline docs/reference/sql/endpoints/cli.asciidoc]: I would change Also -> Moreover or Furthermore

File: docs/reference/sql/endpoints/cli.asciidoc [hunk 2]
```diff
@@ -47,3 +43,27 @@ James S.A. Corey |Leviathan Wakes     |561            |1306972800000
 --------------------------------------------------
 // TODO it'd be lovely to be able to assert that this is correct but
 // that is probably more work then it is worth right now.
+
+The jar containing the SQL CLI is a stand alone Java application and
+the scripts just launch it. You can move it around to other machines
+without having to install Elasticsearch on them. Without the already
+provided script files, you can use a command similar to the following
+to start the SQL CLI. A Java version not earlier than the one used to
```
Comment (matriv) [inline docs/reference/sql/endpoints/cli.asciidoc]: Couldn't be compiled with Java 13 and run with Java 12 (for earler 7.x versions) ?

File: docs/reference/sql/endpoints/cli.asciidoc (hunk 1 above)
Comment (matriv) [inline docs/reference/sql/endpoints/cli.asciidoc]: Maybe also provide an example of the version? like `7.5.2` or something like that.

**Review by matriv — COMMENTED** [review]
Nice addition! Left some comments.

**Review by matriv — APPROVED** [review]
LGTM!

**Comment by bpintea** [comment]
@astefan, we might need to remove the `./` in a new PR.

**Pull request merged**